metadata-models/src/main/pegasus/com/linkedin/ml/metadata/EvaluationData.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.DatasetUrn
+
+/**
+ * All referenced datasets would ideally point to any set of documents that provide visibility into the source and composition of the dataset.
+ */
+record EvaluationData {
```

Comment:
Should the evaluation results be included in the model card?

Reply:
Per the model cards paper - https://arxiv.org/pdf/1810.03993.pdf yes it is part of the Model card.

For ex: If we look at the smile detection model card it gives a complete understanding of the model using also the inputs from Evaluation data.

Reply:
@zimengy `QuantitativeAnalyses` aspect provides the results of evaluating the model if that answers your question.